### ISSUE TITLE
The change of openselfsup to mmselfsup

### ISSUE THREAD

**Issue by JasonRichard** [GAIA-vision/GAIA-ssl#1]
Hi !

[Gaiavison ](https://github.com/GAIA-vision/GAIA-cv) is compatible with mmcv >= 1.2.7 & mmcv < 1.3.0
while openselfsup branch in [mmselfsup](https://github.com/open-mmlab/mmselfsup) installs with default mmcv 1.0.3, and [mmselfsup's](https://mmselfsup.readthedocs.io/en/latest/install.html) document suggests a mmcv version of mmcv-full >= 1.3.16.
I cannot find a mmcv version compatible with both openselfsup and gaiavision.
In other words, I cannot find coresponding openselfsup and gaiavision compatible with mmcv 1.3.0.
Can you provide a detail instruction?

Best

**Comment by NickChang97** [comment]
I am sorry for that. I will put more detailed instructions and take the openself (previous branch of mmselfsup) into this repo.

**Comment by NickChang97** [comment]
Codes of GAIA-vision and GAIA-ssl are alredy updated. You can download the new codes. If you encounter something unexpected, please let me know.

**Comment by tommiekerssies** [comment]
Why is GAIA-ssl not compatible with the latest mmselfsup?

**Comment by NickChang97** [comment]
> Why is GAIA-ssl not compatible with the latest mmselfsup?

Sorry for this, because mmselfsup update much compared to its previous openselfsup version in this year. It's later than CVPR2022, so we don't support it.
